ParallelPrimitives/RadixSort.cpp

```diff
@@ -6,6 +6,9 @@
 #include <iostream>
 #include <numeric>
 
+#include <ParallelPrimitives/cache/Kernels.h>
```

Comment:
These also needs to be conditionally included based on `ORO_PP_LOAD_FROM_STRING`,  I suppose ?
They do not exist unless `bakeKernel.sh` is run.  That flag is optional so it's not always guarantee to be run.

Reply:
oh yes a good catch.